inst/tutorials/treatment/treatment.Rmd

```diff
@@ -0,0 +1,197 @@
+2.  Dihydroartemisinin piperaquine (`DHA_PQP_params`)
+3.  Sulfadoxine pyrimethamine (`SP_AQ_params`)
+
+We can view these parameter sets by simply calling them. Here are the parameters for artemether lumefantrine:
+
+```{r view-inbuilt-antimalarial-parameters, exercise = TRUE, exercise.eval = FALSE}
+
+# View the artemether lumefantrine (AL) parameter set:
+AL_params
+
+```
+
+The parameters in these sets are:
+
+1.  drug_efficacy: probability that treatment with the drug successfully clears the infection
+2.  drug_rel_c: The onward infectiousness of a treated individual to a mosquito relative to an untreated individual
+3.  drug_prophylaxis_shape: Shape parameter for a Weibull distribution describing an individuals prophylactic protection over time following treatment
+4.  drug_prophylaxis_scale: Scale parameter for a Weibull distribution describing an individuals prophylactic protection over time following treatment
+
+```{r view-drug-params, exercise = TRUE}
+
+# View the parameters for DHA-PQP and SP-AQ:
+
+
+```
+
+```{r drug-params-quiz, exercise = TRUE}
+
+quiz(
+  question("The efficacy of DHA-PQP in the DHA_PQP_params set is 0.95000",
+    answer("TRUE", correct = TRUE),
+    answer("FALSE")
+  )
+)
+
+```
+
+To simulate the use of a drug in `malariasimulation`, we load them into the parameters list. This is done using the `set_drugs()` function. Note that the drug parameters (e.g. `AL_params`) must be passed to `get_drugs()` as a list.
+
+```{r using-set-drugs, exercise = TRUE, eval = TRUE}
+
+# Establish a list of malariasimulation parameters:
+parameters <- get_parameters() 
+
+# Load the parameters for artemether lumefantrine into the parameters
+parameters <- set_drugs(parameters = parameters, drugs = list(AL_params))
+
+# View the parameters for DHA-PQP and SP-AQ:
+parameters$drug_efficacy
+
+```
+
+The final line of the chunk above shows us how we can view the drug parameters currently loaded the parameters list once they have been appended. When simulating interventions that use antimalarial drugs (e.g. clinical treatment), this can provide a useful check to ensure you have loaded the drug parameters correctly. In the chunk below, view the `drug_rel_c` parameter:
+
+```{r view-drug-in-params, exercise = TRUE}
+
+# Establish a list of malariasimulation parameters:
+parameters <- get_parameters() 
+
+# Load the parameters for artemether lumefantrine into the parameters
+parameters <- set_drugs(parameters = parameters, drugs = list(AL_params))
+
+# View the drug_rel_c parameter for DHA-PQP:
+
+
+```
+
+## 3. Setting clinical treatment
+
+Now that we know how to load antimalarial drug parameters we can specify how they will be administered to the population. In malariasimulation, we parameterise clinical treatment campaigns using the `set_clinical_treatment()` function (run `?set_clinical_treatment()` to view the help file for the function). `set_clinical_treatment()` requires four arguments:
```

Comment:
I'd not call this a campaign, as that might mean something quite specific, and not usually associated with tx

Reply:
OK good to know! Have removed campaigns